lib/ziti-tunnel/ziti_tunnel.c

```diff
@@ -283,23 +240,9 @@ address_t *parse_address(const char *hn_or_ip_or_cidr, dns_manager *dns) {
     }
 
     if (ipaddr_aton(addr->str, &addr->ip) == 0) {
-        addr->is_hostname = true;
```

Comment:
is there any reason to keep `is_hostname` now?

Reply:
removing